src/entities/Cinema.java

```diff
@@ -76,10 +76,9 @@ public Seat getSeat(String title, int index) {
 	return this.seats.get(title).get(index);  
     }
   
-    public void printLayout(Movie movie, LocalDate showtime) { //prints layout of the Cinema based on available seats for the Movie at the particular showtime
+    public void printLayout(String title, LocalDate showtime) { //prints layout of the Cinema based on available seats for the Movie at the particular showtime
```

Comment:
Changed from object to String since the method in makeBooking already passes in the title.